python_script/meter-health/meter_health.py

```diff
@@ -0,0 +1,119 @@
+import requests
```

Comment:
Not a requirement, but you should add: `# -*- coding: utf-8 -*-` as the very first line of each script. It prevents python from crashing when multibyte characters are used in the code.